routers/comments.py

```
@@ -1,37 +1,171 @@
+COMMENT_PAGE_SIZE = 10
 
 router = APIRouter(
     tags=["COMMENTS"],
 )
 
 
-# List all comments for a post
-@router.get("/posts/{post_id}/comments")
-async def get_comments(post_id: PostId, page: Page):
-    return {"success": "get_comments"}
+def _verify_post_exists(post_id: PostId):
```

Ruff (0.14.13)

Missing return type annotation for private function `_verify_post_exists`

Add return type annotation: `None`

(ANN202)